en/products/vpn/shared.ftl

```diff
@@ -0,0 +1,77 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/
+
+vpn-shared-product-name = { -brand-name-mozilla-vpn }
+vpn-shared-subscribe-link = Get { -brand-name-mozilla-vpn }
+vpn-shared-waitlist-link = Join the Waitlist
+vpn-shared-sign-in-link = Already a subscriber?
+vpn-shared-available-countries = We currently offer { -brand-name-mozilla-vpn } in the US, the UK, Canada, New Zealand, Singapore and Malaysia.
```

Comment:
The list of countries - the style is not consistent. Either add a comma to all three before `and CountryName` or remove it.